thirdeye-spi/src/main/java/ai/startree/thirdeye/spi/api/AnomalyApi.java

```diff
@@ -274,12 +273,4 @@ public AnomalyApi setAnomalyLabels(
     return this;
   }
 
-  public AuthorizationConfigurationApi getAuth() {
```

Comment:
Not sure about the API change. How does the UI know that the user can't access this anomaly?
```
example: https://localhost/anomalies/72410/view
```

We may still need to support this as is. And that's probably not a bad idea. thoughts?

Reply:
Ya, I think that makes sense. I'm just not sure how to populate the auth stuff from the dto -> api mapper. It seems like I would have to include the authorization manager to resolve the auth config, which feels kinda heavy. 

🤔 It might be worth rethinking the api shape for auth anyways, but I'll leave the auth container in api for now.